code/ai/aibig.cpp

```diff
@@ -1041,7 +1158,10 @@ void ai_big_chase()
 		float dist_normal_to_enemy;
 
 		if (vm_vec_mag_squared(&aip->big_attack_surface_normal) > 0.9) {
-			dist_normal_to_enemy = fl_abs((dist_to_enemy * vm_vec_dot(&vec_to_enemy, &aip->big_attack_surface_normal)));
+			if (aip->ai_profile_flags[AI::Profile_Flags::Improved_subsystem_attack_pathing])
+				dist_normal_to_enemy = fl_abs((dist_to_enemy / vm_vec_dot(&vec_to_enemy, &aip->big_attack_surface_normal)));
+			else // I'm fairly the retail this retail code VVV meant to use / instead of * 
```

Comment:
Very minor, but this comment is not very clear. Did you mean `I'm fairly certain the retail code VVV meant to use / instead of *`?

Reply:
oops, yes

Reply:
Cool. Other than that minor comment cleanup I'd say this looks good to go.